src/bin/ch-remote.rs

```diff
@@ -419,6 +609,18 @@ struct TopLevel {
     #[argh(option, long = "api-socket")]
     /// HTTP API socket path (UNIX domain socket)
     api_socket: Option<String>,
+
+    #[argh(option, long = "dbus-service-name")]
```

Comment:
Should these be gated by `dbus_api` too? But then sprinkling that everywhere looks ugly. I'm a bit two-minded here. :-/

Reply:
I believe it is reasonable to conditionally compile it based on the `dbus_api` flag. I will investigate if there is a better approach to implement this that allows us to conditionally compile these without making things messy.

Reply:
Hey, @liuw, I've updated the PR to remove the D-Bus related code from `ch-remote` if the `dbus_api` feature flag isn't present. This change increases the LOC, but I believe it's justified because adding cfg attributes throughout the code in the previous implementation would be too ugly as you also noted. What do you think about the new implementation?

Reply:
Any updates on the status of this review? @liuw Thank you.